core/vm/contracts.go

```diff
@@ -121,6 +122,12 @@ var PrecompiledContractsBLS = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{18}): &bls12381MapG2{},
 }
 
+// PrecompiledContractsP256Verify contains the precompiled Ethereum
+// contract specified in EIP-7212. This is exported for testing purposes.
+var PrecompiledContractsP256Verify = map[common.Address]PrecompiledContract{
+	common.BytesToAddress([]byte{19}): &p256Verify{},
+}
+
```

Comment:
The implementation address updated to `0x0b` due to the [EIP specification](https://eips.ethereum.org/EIPS/eip-7212#specification) in my branch with [this commit](https://github.com/ulerdogan/go-ethereum/commit/99c44aec7e190ca4c027cb1714837bc301b2a368), but have not synced into this branch as it's closed.

```
// PrecompiledContractsP256Verify contains the precompiled Ethereum
// contract specified in EIP-7212. This is exported for testing purposes.
var PrecompiledContractsP256Verify = map[common.Address]PrecompiledContract{
	common.BytesToAddress([]byte{0x0b}): &p256Verify{},
}
```

Reply:
The implementation address updated to `0x100` again with [this commit](https://github.com/ulerdogan/go-ethereum/commit/b5a6bb06d41b53586579822bdb2807cc8418d367) as the EIP moved into the RIPs repo and this range has been attached for the RIP precompiles.

```
// PrecompiledContractsP256Verify contains the precompiled Ethereum
// contract specified in RIP-7212. This is exported for testing purposes.
var PrecompiledContractsP256Verify = map[common.Address]PrecompiledContract{
	common.BytesToAddress([]byte{0x01, 0x00}): &p256Verify{},
}
```